src/index.tsx

```diff
@@ -10,9 +9,7 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
```

Comment:
What does that change and can we avoid it?

Reply:
"[Remove StrickMode it caused the application to render multiple times.](https://github.com/doskachok/magellan/pull/76/commits/225bb7e483909438395165a2b1dbbad5fa5d8b41)"
In this case it executed 2/3 toasts about successful registration.

Reply:
Well, in all other cases it seemed to work properly, what’s so special about hit this push notification?

Reply:
It works the same in other cases(several renders)